client/src/components/timeline-create-entry-components/contacted-entry-component/index.jsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react";
+import {
+  Card,
+  Box,
+  Avatar,
+  CardContent,
+  TextField,
+  Button,
+  Typography,
+  Divider,
+  Select,
+  MenuItem,
+} from "@mui/material";
+import PhoneIphoneIcon from "@mui/icons-material/PhoneIphone";
+// import CloseIcon from '@mui/icons-material/Close';
+
+function ContactedEntryComponent() {
+  const [selectedValue, setSelectedValue] = useState("Employer");
+  const [notes, setNotes] = useState("");
+
+  const handleValueChange = (e) => {
+    setSelectedValue(e.target.value);
+  };
+
+  const handleNoteChange = (e) => {
+    setNotes(e.target.value);
+  };
+
+  return (
+    <Card sx={{ height: 791, width: 391 }}>
+      <CardContent>
+        <Typography variant="h5">Activity Timeline</Typography>
+        <Box sx={{ display: "flex", marginTop: "12px" }}>
+          <Box>
+            <Avatar
+              sx={{
+                bgcolor: "rgba(229, 53, 104, 1)",
+                width: "30px",
+                height: "30px",
+              }}
+            >
+              <PhoneIphoneIcon sx={{ width: "18px", height: "18px" }} />
+            </Avatar>
+          </Box>
+          <Box
+            sx={{
+              display: "flex",
+              justifyContent: "center",
+              alignItems: "center",
+              marginLeft: "16px",
+            }}
+          >
+            <Typography>Contacted Employer</Typography>
```

Comment:
If we can conditionally render this based on if it will be used for an employer or a client that would be great.

Since we do need a page to add contacting a client to the timeline